modules/layers/src/column-layer/column-layer.js

```diff
@@ -115,28 +125,50 @@ export default class ColumnLayer extends Layer {
     });
   }
 
-  _getModel(gl) {
-    return new Model(
-      gl,
-      Object.assign({}, this.getShaders(), {
-        id: this.props.id,
-        geometry: this.getGeometry(this.props.diskResolution),
-        isInstanced: true,
-        shaderCache: this.context.shaderCache
-      })
-    );
+  _getModels(gl) {
+    const {id, filled, extruded, diskResolution} = this.props;
+
+    let filledModel;
+    let strokeModel;
+
+    if (filled) {
+      filledModel = new Model(
+        gl,
+        Object.assign({}, this.getShaders(), {
+          id: `${id}-top`,
+          geometry: this.getGeometry(diskResolution),
+          isInstanced: true,
+          shaderCache: this.context.shaderCache
+        })
+      );
+    }
+    if (extruded) {
```

Comment:
`wireframe`. We do want stroke in non-extruded mode.

Reply:
@kaushikb11 

this PR is for enabling `wireframe` feature for `ColumnLayer`. So the conditional drawing here should be 

```

if (filled) {
  fillModel = ...
}

if (wireframe) {
  wireframeModel = ...
}


```